pycbc/strain/strain.py

```diff
@@ -724,6 +700,10 @@ def insert_strain_option_group_multi_ifo(parser, gps_times=True):
                     metavar="IFO:VAL", default=1.,
                     help="Multiple injections by this factor "
                          "before injecting into the data.")
+    data_reading_group_multi.add_argument("--injection-sample-rate",
+                    type=float, nargs="+", action=MultiDetOptionAction,
+                    metavar="IFO:VAL",
+                    help="Sample rate to generate injection with")
```

Comment:
Thanks for not forgetting this branch! I still dislike having to give each option twice, but cannot see a better solution (although I wonder if we could just use the multi-ifo group even in the single-ifo case without having to make single-ifo code more complicated ...)